java/runtime-common/src/main/java/org/ray/core/RayRuntime.java

```diff
@@ -260,50 +259,27 @@ public UniqueID getCurrentTaskNextPutId() {
     return objectStoreProxy.wait(waitfor, numReturns, timeout);
   }
 
-  @Override
-  public RayObjects call(UniqueID taskId, Callable funcRun, int returnCount, Object... args) {
-    return worker.rpc(taskId, funcRun, returnCount, args);
-  }
-
   @Override
   public RayObjects call(UniqueID taskId, Class<?> funcCls, Serializable lambda, int returnCount,
-                         Object... args) {
-    return worker.rpc(taskId, UniqueID.nil, funcCls, lambda, returnCount, args);
-  }
-
-  @Override
-  public <R, RIDT> RayMap<RIDT, R> callWithReturnLabels(UniqueID taskId, Callable funcRun,
-                                                        Collection<RIDT> returnIds,
-                                                        Object... args) {
-    return worker.rpcWithReturnLabels(taskId, funcRun, returnIds, args);
+      Object... args) {
+    return worker.rpc(taskId, funcCls, lambda, returnCount, args);
   }
 
   @Override
   public <R, RIDT> RayMap<RIDT, R> callWithReturnLabels(UniqueID taskId, Class<?> funcCls,
-                                                        Serializable lambda, Collection<RIDT>
-                                                          returnids,
-                                                        Object... args) {
+      Serializable lambda, Collection<RIDT>
+      returnids,
```

Comment:
nit: put `Collection<RIDT>` and `returnids` in the same line?

Reply:
ok